<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="controlsTab">
       <attribute name="title">
        <string>Ui</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="sensorsTab">
       <attribute name="title">
        <string>Sensors</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7"/>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Uuid gen</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QWidget" name="widget_7" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_8" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="uuidGenBtn">
                <property name="text">
                 <string>Generate</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Uuid (text view)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="uuidViewText">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Uuid compressed hex value (for MC sketches)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="uuidCompressedHex">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>460</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>402</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Ui</string>
    </property>
    <addaction name="openUiXmlAction"/>
    <addaction name="openUiJsonAction"/>
    <addaction name="saveUiXmlAction"/>
    <addaction name="saveUiJsonAction"/>
    <addaction name="copyUiXmlAsCVarAction"/>
    <addaction name="copyUiJsonAsCVarAction"/>
   </widget>
   <widget class="QMenu" name="menuSensors">
    <property name="title">
     <string>Sensors</string>
    </property>
    <addaction name="openSensorsXmlAction"/>
    <addaction name="openSensorsJsonAction"/>
    <addaction name="saveSensorsXmlAction"/>
    <addaction name="saveSensorsJsonAction"/>
    <addaction name="copySensorsXmlAsCVarAction"/>
    <addaction name="copySensorsJsonAsCVarAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSensors"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openUiXmlAction">
   <property name="text">
    <string>Open xml</string>
   </property>
  </action>
  <action name="saveUiJsonAction">
   <property name="text">
    <string>Save as json</string>
   </property>
  </action>
  <action name="saveUiXmlAction">
   <property name="text">
    <string>Save as xml</string>
   </property>
  </action>
  <action name="copyUiXmlAsCVarAction">
   <property name="text">
    <string>Copy xml to buffer as C var</string>
   </property>
  </action>
  <action name="copyUiJsonAsCVarAction">
   <property name="text">
    <string>Copy json to buffer as C var</string>
   </property>
  </action>
  <action name="openUiJsonAction">
   <property name="text">
    <string>Open json</string>
   </property>
  </action>
  <action name="openSensorsXmlAction">
   <property name="text">
    <string>Open xml</string>
   </property>
  </action>
  <action name="openSensorsJsonAction">
   <property name="text">
    <string>Open json</string>
   </property>
  </action>
  <action name="saveSensorsXmlAction">
   <property name="text">
    <string>Save as xml</string>
   </property>
  </action>
  <action name="saveSensorsJsonAction">
   <property name="text">
    <string>Save as json</string>
   </property>
  </action>
  <action name="copySensorsXmlAsCVarAction">
   <property name="text">
    <string>Copy xml to buffer as C var</string>
   </property>
  </action>
  <action name="copySensorsJsonAsCVarAction">
   <property name="text">
    <string>Copy json to Buffer as C var</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
